pysat/instruments/pysat_testing.py

```diff
@@ -73,64 +70,109 @@
-meta['dummy2'] = {'units':'', 'long_name':'dummy2'}
-meta['dummy3'] = {'units':'', 'long_name':'dummy3'}
-meta['dummy4'] = {'units':'', 'long_name':'dummy4'}
-meta['string_dummy'] = {'units':'', 'long_name':'string_dummy'}
-meta['unicode_dummy'] = {'units':'', 'long_name':'unicode_dummy'}
-meta['int8_dummy'] = {'units':'', 'long_name':'int8_dummy'}
-meta['int16_dummy'] = {'units':'', 'long_name':'int16_dummy'}
-meta['int32_dummy'] = {'units':'', 'long_name':'int32_dummy'}
-meta['int64_dummy'] = {'units':'', 'long_name':'int64_dummy'}
+meta['longitude'] = {'units': 'degrees', 'long_name': 'Longitude'} 
+meta['latitude'] = {'units': 'degrees', 'long_name': 'Latitude'} 
+meta['dummy1'] = {'units': '', 'long_name': 'dummy1'}
+meta['dummy2'] = {'units': '', 'long_name': 'dummy2'}
+meta['dummy3'] = {'units': '', 'long_name': 'dummy3'}
+meta['dummy4'] = {'units': '', 'long_name': 'dummy4'}
+meta['string_dummy'] = {'units': '', 'long_name': 'string_dummy'}
+meta['unicode_dummy'] = {'units': '', 'long_name': 'unicode_dummy'}
+meta['int8_dummy'] = {'units': '', 'long_name':' int8_dummy'}
+meta['int16_dummy'] = {'units': '', 'long_name': 'int16_dummy'}
+meta['int32_dummy'] = {'units': '', 'long_name': 'int32_dummy'}
+meta['int64_dummy'] = {'units': '', 'long_name': 'int64_dummy'}
 
 
         
 def init(self):
-    self.new_thing=True        
-                
+    """ Initialization function
+
+    Parameters
+    ----------
+    file_date_range : (pds.date_range)
+        Optional keyword argument that specifies the range of dates for which
+        test files will be created
+
+    """
+    self.new_thing=True
+
+    if 'file_date_range' in self.kwargs:
+        # set list files routine to desired date range
+        # attach to the instrument object
+        self._list_rtn = functools.partial(list_files, \
+                                file_date_range=self.kwargs['file_date_range'])
+        self.files.refresh()
+
 def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
-         sim_multi_file_left=False, root_date=None):
+         sim_multi_file_left=False, root_date=None, file_date_range=None):
+    """ Loads the test files
+
+    Parameters
+    ----------
+    fnames : (list)
+        List of filenames
+    tag : (str or NoneType)
+        Instrument tag (accepts '' or a number (i.e., '10'), which specifies
+        the number of times to include in the test instrument)
+    sat_id : (str or NoneType)
+        Instrument satellite ID (accepts '')
+    sim_multi_file_right : (boolean)
+        Adjusts date range to be 12 hours in the future or twelve hours beyond
+        root_date (default=False)
+    sim_multi_file_left : (boolean)
+        Adjusts date range to be 12 hours in the past or twelve hours before
+        root_date (default=False)
+    root_date : (NoneType)
+        Optional central date, uses test_dates if not specified.  (default=None)
+    file_date_range : (pds.date_range or NoneType)
```

Comment:
Did the code object without the file_date_range keyword in load?

Reply:
by code I mean pysat

Reply:
Yes it did.

Reply:
Good. I was worried that having keywords used by both init and load could lead to potential collisions or confusion.